control/tests/test_api_question_file.py

```diff
@@ -98,8 +110,8 @@ def test_audited_cannot_update_question_file_from_draft_questionnaire():
         "question": question_file.question.id + 1
     }
 
-    # method not allowed
-    assert update_question_file(audited.user, payload).status_code == 405
+    # Forbidden
+    assert update_question_file(audited.user, payload).status_code == 403
```

Comment:
j'avoue que j'ai du mal à savoir quand on doit faire "Method not allowed" ou "Forbidden" ou "Not found", c'est un peu subtil... Pour le moment du moment que c'est interdit c'est deja pas mal :) 

(si on veut ergoter, il peut y avoir des problemes qd le code d'erreur est different selon que la ressource existe ou pas. Parce que ca peut reveler l'existence d'une ressource. Mais bon, peut etre qu'on a assez de choses a faire sans chipoter.)

Reply:
ok. J'ai l'impression que ça ramène à la conversation "est ce qu'on doit tester method not allowed". On dit qu'on avance avec ce qu'on a là puisque ça bouche les trous.